.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    shell: bash # Windows' default shell is PowerShell
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false # when true, one job's failure will cancel other jobs
+      matrix:
+        os: [ ubuntu-16.04, macOS-latest, windows-latest ]
+    steps:
+      - name: Add "$HOME/.local/bin/" to PATH
+        run: echo "$HOME/.local/bin/" >> $GITHUB_PATH
+
+      # NB: We install gnu-tar because BSD tar is buggy on Github's MacOS machines,
+      # and it breaks the cache: https://github.com/actions/cache/issues/403
+      - name: Install GNU tar (macOS)
+        if: runner.os == 'macOS'
+        run: |
+          brew install gnu-tar
+          echo "/usr/local/opt/gnu-tar/libexec/gnubin" >> $GITHUB_PATH
+
+      - uses: actions/checkout@v2
+
+      # Caching
+      #
+      # `haskell/actions/setup` will overwrite the STACK_ROOT variable
```

Comment:
I think we can remove this workaround since this was fixed upstream? https://github.com/haskell/actions/issues/23

Reply:
Yes, I believe so.